tools/functional/conftest.py

```diff
@@ -1,6 +1,7 @@
 import tempfile
 import time
 import uuid
+from contextlib import contextmanager
```

Comment:
oops this one needs to go, will remove it asap.

Reply:
```suggestion
```